_guides/005-operations.md

```diff
@@ -37,9 +37,9 @@ sections:
     description: StorageOS licensing
     module: operations/licensing
   -
-    title: Version update
-    description: StorageOS update
-    module: operations/updates
+    title: Version upgrade
```

Comment:
Suggest we capitalise words in headings - 'StorageOS Upgrade' or 'Upgrading StorageOS'